### ISSUE TITLE
Difference in Dataset Path Behavior Between YOLOv5 and This Repo

### ISSUE THREAD

**Issue by tino926** [ultralytics/ultralytics#17552]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

The automatically created datasets folder for training data is not located at the same path as YOLOv5. 

Previously, when I used YOLOv5 to train with the official `coco.yaml`, it downloaded the COCO dataset and placed it in a folder next to the working directory. For example, if my working directory was `/my_projects/yolov5_working`, the COCO dataset would be downloaded to `/my_projects/datasets/coco`.

Now, I am using YOLOv11 (installed using git clone method). When training with the official `coco.yaml`, the COCO dataset is downloaded into a `datasets` folder inside the working directory. For instance, if my working directory is `/my_projects/ultralytics_working`, the COCO dataset is downloaded to `/my_projects/ultralytics_working/datasets`. 

Is this the correct behavior, or am I doing something wrong?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @tino926, thank you for your interest in Ultralytics 🚀! This is an automated response to help guide you while an Ultralytics engineer reviews your question more closely. 

We recommend first visiting the [Docs](https://docs.ultralytics.com) for details on YOLOv11 usage. If you're noticing differences in dataset paths between YOLOv5 and the current repo, it could be due to changes in the default configurations. For a detailed comparison, check the [Model ZOO](https://docs.ultralytics.com/yolov5/).

Could you please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to further illustrate the issue? This would help us debug the path behavior more effectively.

If you have more general questions, feel free to share any dataset examples or log outputs as you verify adherence to our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://ultralytics.com/discord) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com). Or dive into threads on our [Subreddit](https://reddit.com/r/ultralytics) to share knowledge with the community.

## Upgrade

To ensure you have the latest features and fixes, upgrade your `ultralytics` package, along with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml), using a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLOv11 can be executed in various verified environments for ease of use and setup, including:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If the badge above is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing. CI tests ensure proper function of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across multiple platforms like macOS, Windows, and Ubuntu on a daily basis and with every code change.

**Comment by Y-T-G** [comment]
> Is this the correct behavior

Yes
